WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -446,4 +455,95 @@ public static void setReaderSwipeToNavigateShown(boolean alreadyShown) {
         setBoolean(UndeletablePrefKey.SWIPE_TO_NAVIGATE_READER, alreadyShown);
     }
 
+    public static ArrayList<Long> getPendingDraftsIgnorePostIdList() {
+        // builds the list from csv
+        ArrayList<Long> idList = new ArrayList<>();
+        String idListString = getString(DeletablePrefKey.PENDING_DRAFTS_NOTIFICATION_IGNORE_LIST, "");
+        if (!TextUtils.isEmpty(idListString)) {
+            List<String> items = Arrays.asList(idListString.split("\\s*,\\s*"));
```

Comment:
Can't this be simplified in ` List<String> items = Arrays.asList(idListString.split(","));` ?